themes/base.theme.bash

```diff
@@ -390,6 +390,27 @@ function hg_prompt_vars() {
 	fi
 }
 
+function node_command_version_prompt() {
+	local node_version
+	if _command_exists node; then
+		node_version="$(node --version 2> /dev/null)"
+		if [[ -n ${node_version} ]]; then
```

Comment:
This check would give a false positive when using version management shims that are common with tools like asdf

Reply:
Could you clarify? What do you mean by a false positive?

if `node --version` returns a non-empty output, doesn't that mean that this is the current node version? If so, why not show it in the prompt?